### PR TITLE
Left bar component menu + AdministratorPanel + Manage Administrator

### PR DESCRIPTION
**Pull request by SerhiiTarasiuk**

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestPages/AdministratorManagePage.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        }
+
+        public void SendKeysToInputPasswordForNewAdministrator(string password)
+        {
+            _inputPasswordForNewAdministrator.SendKeys(password);
+        }
+
+        public void SendKeysToInputPhoneNumberForNewAdministrator(string phonenumber)
+        {
+            _inputPhoneNumberForNewAdministrator.SendKeys(phonenumber);
```
Comment (AhnessP) [inline EasyRestProjectNetTeam2/EasyRestPages/AdministratorManagePage.cs:69]: please rename  phoneNumber for  better readability

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/LeftBarMenuComponent.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        public void ClickOwnersLeftBarButton()
+        {
+            _ownersLeftBarButton.Click();
+        }
+
+        public void ClickModeratorsLeftBarButton()
+        {
+            _moderatorsLeftBarButton.Click();
+        }
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/LeftBarMenuComponent.cs]: please remove empty lines

File: EasyRestProjectNetTeam2/EasyRestPages/AdministratorManagePage.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        [FindsBy(How = How.XPath, Using = "//p[text()='Mail is required']")]
+        private IWebElement _inputEmailValidationWarning;
+
+        [FindsBy(How = How.XPath, Using = "//p[text()='Password is required']")]
+        private IWebElement _inputPasswordValidationWarning;
+
+        [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
+        private IWebElement _inputPhoneNumberValidationWarning;
+
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdministratorManagePage.cs:46]: please remove useless lines, but leave one line between locator and method

File: EasyRestProjectNetTeam2/EasyRestPages/AdministratorPanelPage.cs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            _acceptedOrderTab.Click();
+        }
+
+        public void ClickAcceptOrderButton()
+        {
+            _acceptOrderButton.Click();
+        }
+
+        public void ClickToGetInfoAboutOrderArrow()    //First one in the list
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdministratorPanelPage.cs]: Is the comment actual for that method?

File: EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs [hunk 5]
```diff
@@ -1,14 +1,19 @@
-﻿using OpenQA.Selenium;
+﻿using EasyRestProjectNetTeam2.EasyRestComponentsObj;
+using OpenQA.Selenium;
 using SeleniumExtras.PageObjects;
 
 namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class MenuPage : BasePage
     {
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs]: please remove the empty line

File: EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs [hunk 6]
```diff
@@ -1,14 +1,19 @@
...
+        public LeftBarComponent LeftBarComponent { get; set; }
         public MenuPage(IWebDriver driver) : base(driver)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs:11]: please add one line between property and constructor for better readability

**Review by dmatsev — CHANGES_REQUESTED** [review]
Please resolve the comments.

**Review by Slim1011 — APPROVED** [review]

**Review by AhnessP — APPROVED** [review]

**Review by dmatsev — APPROVED** [review]

**Pull request merged**